### PR TITLE
Extended sign up form, now contains fields needed for profile

### PR DESCRIPTION
**Pull request by manoadamro**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,27 +1,24 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Acebook</title>
-    <%= csrf_meta_tags %>
+<head>
```
Comment (neoeno) [inline app/views/layouts/application.html.erb]: There are a lot of changes in this file — for instance the `title` tag gets removed. Can we change it so the changes are just exactly what you want?

File: spec/controllers/posts_controller_spec.rb [hunk 2]
```diff
@@ -19,5 +20,10 @@
       get :index
       expect(response).to have_http_status(200)
     end
+
+    scenario "renders posts template" do
```
Comment (neoeno) [inline spec/controllers/posts_controller_spec.rb]: Use `it` in unit tests.

Comment (neoeno) [reply spec/controllers/posts_controller_spec.rb]: (The strategy here is noticing what it does above, and staying consistent)

File: spec/features/clearance/visitor_signs_up_spec.rb [hunk 3]
```diff
@@ -11,19 +11,19 @@
   end
 
   scenario "with valid email and password" do
-    sign_up_with "valid@example.com", "password"
+    sign_up_with "valid@example.com", "password", "Michael", "I am Michael", "01/01/1970", "yoga, crack", "Earth"
```
Comment (neoeno) [inline spec/features/clearance/visitor_signs_up_spec.rb]: Doesn't feel like all of these attributes are important for this test. Can you make them optional?

**Review by neoeno — CHANGES_REQUESTED** [review]
Looking good — just a few things!

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (manoadamro) [reply app/views/layouts/application.html.erb]: not sure how this happened, wasn't intending to change anything in here. have reverted back to master

File: spec/controllers/posts_controller_spec.rb (hunk 2 above)
Comment (manoadamro) [reply spec/controllers/posts_controller_spec.rb]: now uses 'it'

File: spec/features/clearance/visitor_signs_up_spec.rb (hunk 3 above)
Comment (manoadamro) [reply spec/features/clearance/visitor_signs_up_spec.rb]: everything except 'email' and 'password' is now optional

**Comment by manoadamro** [comment]
I think it's time to close this one

**Pull request closed without merging**